packages/website/src/components/menu.tsx

```diff
@@ -70,15 +69,16 @@ type MenuLinkProps = {
 export const MenuLink = ({ link }: MenuLinkProps) => {
   const setIsMenuOpen = useSetAtom(menuAtom);
 
-  let Element: any = 'a'; // Switch to <Link>: ElementType
-  const props: ComponentPropsWithoutRef<'a'> = {};
-
-  props.href = link.href;
+  let Element: any = Link;
+  const props: any = {};
 
   if (link.href.startsWith('http')) {
     Element = 'a';
     props.target = '_blank';
     props.rel = 'noopener noreferrer';
+    props.href = link.href;
+  } else {
+    props.to = link.href;
```

Comment:
`to` is a required parameter, and this will cause error if it's empty

https://github.com/dai-shi/waku/blob/59a2c606f8d2adf98475f25d382f65e5644cb1a0/packages/waku/src/router/client.ts#L95-L97

Reply:
No never mind, I missed your `Element = 'a';` on the top

Reply:
But reduce `any` types would help for reading

Reply:
Yeah, I agree. It's not very readable.